<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSMSClass</class>
 <widget class="QMainWindow" name="CSMSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSMS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/icon .ico</normaloff>res/icon .ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>721</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="managePage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/prefix1/res/back09.jpg);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>品牌车管理</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>351</width>
        <height>361</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/prefix1/res/back12.jpg);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>347</width>
         <height>293</height>
        </rect>
       </property>
       <attribute name="label">
        <string>出售车辆</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>厂家：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>品牌：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>报价：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>金额：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="sellFactoryComboBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="sellBrandComboBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="sellPriceLineEdit">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sellNumSpinBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>150</y>
          <width>46</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="sellSumLineEdit">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>190</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>110</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>万</string>
        </property>
       </widget>
       <widget class="QLabel" name="sellLastNumLabel">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>150</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>剩余数量：000</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>190</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>万</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sellOkBtn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>250</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sellCancelBtn">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>250</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>347</width>
         <height>293</height>
        </rect>
       </property>
       <attribute name="label">
        <string>新车入库</string>
       </attribute>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>厂家：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>品牌：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="inNumSpinBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>46</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addBtn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancleBtn2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>220</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
       <widget class="QComboBox" name="inFactoryComboBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="inBrandComboBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>87</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QListWidget" name="dailyList">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>351</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chartPage">
     <widget class="QPushButton" name="updateBtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>更新显示</string>
      </property>
     </widget>
     <widget class="QComboBox" name="factoryComboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>711</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/prefix1/res/back10.jpg);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>销售统计</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>销售管理(&amp;S)</string>
    </property>
    <addaction name="manageAction"/>
    <addaction name="chartAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_C">
    <property name="title">
     <string>设置(&amp;E)</string>
    </property>
    <addaction name="changePasswordAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_a">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutQtAction"/>
    <addaction name="aboutSoftwareAction"/>
    <addaction name="separator"/>
    <addaction name="helpAction"/>
   </widget>
   <addaction name="menu_S"/>
   <addaction name="menu_C"/>
   <addaction name="menu_a"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="manageAction"/>
   <addaction name="chartAction"/>
   <addaction name="separator"/>
   <addaction name="changePasswordAction"/>
   <addaction name="separator"/>
   <addaction name="aboutSoftwareAction"/>
   <addaction name="aboutQtAction"/>
   <addaction name="helpAction"/>
   <addaction name="separator"/>
   <addaction name="quitAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="manageAction">
   <property name="icon">
    <iconset resource="csms.qrc">
     <normaloff>:/new/prefix1/res/edit.png</normaloff>:/new/prefix1/res/edit.png</iconset>
   </property>
   <property name="text">
    <string>品牌车管理(&amp;M)</string>
   </property>
  </action>
  <action name="chartAction">
   <property name="icon">
    <iconset resource="csms.qrc">
     <normaloff>:/new/prefix1/res/chart.png</normaloff>:/new/prefix1/res/chart.png</iconset>
   </property>
   <property name="text">
    <string>销售统计(&amp;C)</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="csms.qrc">
     <normaloff>:/new/prefix1/res/switch.png</normaloff>:/new/prefix1/res/switch.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="changePasswordAction">
   <property name="icon">
    <iconset resource="csms.qrc">
     <normaloff>:/new/prefix1/res/changepass.png</normaloff>:/new/prefix1/res/changepass.png</iconset>
   </property>
   <property name="text">
    <string>修改密码(&amp;P)</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="icon">
    <iconset resource="csms.qrc">
     <normaloff>:/new/prefix1/res/Qt.png</normaloff>:/new/prefix1/res/Qt.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="aboutSoftwareAction">
   <property name="icon">
    <iconset resource="csms.qrc">
     <normaloff>:/new/prefix1/res/icon .ico</normaloff>:/new/prefix1/res/icon .ico</iconset>
   </property>
   <property name="text">
    <string>关于软件(&amp;A)</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="icon">
    <iconset resource="csms.qrc">
     <normaloff>:/new/prefix1/res/help.png</normaloff>:/new/prefix1/res/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助(&amp;H)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="csms.qrc"/>
 </resources>
 <connections/>
</ui>
